python/src/tests/tasks/test_embedding.py

```diff
@@ -49,35 +49,8 @@ def test_throws_on_missing_parameters(self):
     def test_works_with_request(self):
         GetEmbedding(self.correct_request_pca)
 
-    def test_pca_edits_object_appropriately(self):
```

Comment:
These are removed because they are tested by the R side, right?

Reply:
actually these are removed because they are not doing anything right now. We couldn't test this in R properly, and that's why we might move the embedding to the pipeline.

Reply:
could we leave them commented out with this explanation? 
or is this 
> we might move the embedding to the pipeline.

written down somewhere else?

Reply:
(I ask because if it is written down somewhere that this will be moved to the pipeline then we can remove this)